src/v3/src/components/Spinner/Spinner.tsx

```diff
@@ -13,25 +13,34 @@
 import { Box, CircularProgress } from '@okta/odyssey-react-mui';
 import { FunctionComponent, h } from 'preact';
 
-const Spinner: FunctionComponent<{
-  label?: string;
-  valueText?: string;
-}> = ({
-  label,
-  valueText,
-}) => (
-  <Box
-    display="flex"
-    flexDirection="column"
-    justifyContent="center"
-    alignItems="center"
-  >
-    <CircularProgress
-      // TODO: OKTA-518793 - replace english string with key once created
-      aria-label={label || 'Loading...'}
-      aria-valuetext={valueText || 'Loading...'}
-    />
-  </Box>
-);
+import { SpinnerElement } from '../../types';
+import { loc } from '../../util';
+
+type SpinnerProps = { dataSe?: string; color?: string; };
+const Spinner: FunctionComponent<SpinnerProps | SpinnerElement> = (
+  props,
+) => {
+  const { dataSe = undefined, color = undefined } = 'type' in props
+    ? {}
+    : props as SpinnerProps;
```

Comment:
I updated the spinner props to allow for usage of this component outside of the scope of the transformer layer. The reason for this is because the Spinner is needed/used in the widget index file which doesn't go through transformers and it would be unnecessary to pass props the way the Layout.tsx file does. 

Additionally, I added the `loc` function directly here because of that reason, it doesnt go through transformers for some areas so it would be unnecessary to add properties the way transformers do.